### PR TITLE
[chore] fix zookeeper supported pipeline types

### PR DESCRIPTION
**Pull request by ericmustin**
**Description:** <Describe what has changed.>

_tl;dr typo fix_

This PR updates the zookeeper receiver docs to reflect the correct supported pipeline. Currently the zookeeper receiver supports the `metrics` pipeline (see: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/1bf261e29c2d9d088cefc71d595b88ed58668883/receiver/zookeeperreceiver/factory.go#L42) however it's labeled as `traces` in the Readme. this PR fixes this typo.

**Link to tracking Issue:** <Issue number if applicable> n/a

**Testing:** <Describe what testing was performed and which tests were added.> n/a

**Documentation:** <Describe the documentation added.> updated readme

### CODE REVIEW

**Comment by ericmustin** [comment]
I don't believe this requires a changelog entry as it's a typo fix. Happy to add if that's preferred

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
Thanks for the fix, @ericmustin. The changelog entry check is skipped if PR name has `[chore]` part in it :)
